content/en/blog/2022-10-14-helm-at-kubecon-na-22.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Helm @ KubeCon + CloudNativeCon NA '22"
+slug: "helm-at-kubecon-na-22"
+authorname: "Karen Chu"
+author: "@karenhchu"
+authorlink: "https://twitter.com/karenhchu"
+date: "2022-10-14"
+---
+
+The Helm maintainers are excited to be headed to KubeCon + CloudNativeCon NA '22 in Detroit, MI a couple of weeks! As always, there will be a few different places you can find us!
+
+## Helm Maintainer Track Talk: [Learn about Helm and Its Ecosystem](https://sched.co/182Ns)
+
+**Date**: Thursday October 27
+
+**Time**: 11:00am - 11:35am EDT
+
+**Location**: Room 410 A
+
+Maintainers [Matt Farina](https://twitter.com/mattfarina), [Karena Angel](https://twitter.com/karenaangell)l, [Scott Rigby](https://twitter.com/r6by), & [Andrew Block](https://twitter.com/sabre1041) will be doing a [Maintainer Track talk](https://sched.co/182Ns) introducing Helm itself and then deep dive into the ecosystem around building packages, along with using Helm packages in clusters. 
+
+
+## Project Pavilion Hours:
+
+**Wed Oct 26**: 15:30 – 20:00 ET
+
+**Thur Oct 27**: 14:00 – 17:30 ET
+
+**Fri Oct 28**: 13:00 – 16:00 ET
+
+**Location**: on the backside of the Fuzz booth, near the LitmusChaos/Flux/Cortex/Falco booths
+
+Our maintainers will be hanging out at our Helm project booth the second half of each day. We'd love to learn from the community how Helm is being used, what pain points exists, and what users are looking for in the future. Help us help you by sharing your thoughts in our [Helm User Survey](https://docs.google.com/forms/d/e/1FAIpQLSeR9fSlWShh49_URhAEPA88JVjlPiz1441CA1B2ySJGZg1dzQ/viewform), even if you're unable to drop by the booth. 
```

Comment:
I clicked on https://docs.google.com/forms/d/e/1FAIpQLSeR9fSlWShh49_URhAEPA88JVjlPiz1441CA1B2ySJGZg1dzQ/viewform and it says EU?

Reply:
Oops. Just updated the title of the Google form. Link is correct though. Thanks for catching that.